Doxyfile

```diff
@@ -0,0 +1,58 @@
+PROJECT_NAME           = Vulkan-Hpp
+SEARCHENGINE           = YES
+GENERATE_TAGFILE       = html/tagfile.xml  # generate a tag file
+DOT_IMAGE_FORMAT       = svg
+DOT_TRANSPARENT        = YES
+INTERACTIVE_SVG        = YES
+EXTRACT_ALL            = YES
+RECURSIVE = YES
+INPUT = vulkan/vulkan.cppm \
+vulkan/vulkan.hpp \
+vulkan/vulkan_enums.hpp \
+vulkan/vulkan_extension_inspection.hpp \
+vulkan/vulkan_format_traits.hpp \
+vulkan/vulkan_funcs.hpp \
+vulkan/vulkan_hash.hpp \
+vulkan/vulkan_hpp_macros.hpp \
+vulkan/vulkan_to_string.hpp \
+vulkan/vulkan_handles.hpp \
+vulkan/vulkan_video.hpp \
+vulkan/vulkan_static_assertions.hpp \
+vulkan/vulkan_raii.hpp \
+vulkan/vulkan_structs.hpp \
+vulkan/vulkan_structs.hpp \
```

Comment:
without VulkanSC to make webpage not unnecessary slower for class list. doxygen crashed on `VkSharedHandle` by encountering an infinite recursion loop when investigating template relations.